apps/namada-interface/src/slices/proposals.ts

```diff
@@ -35,7 +35,7 @@ export const fetchProposals = createAsyncThunk<
   void,
   { state: RootState }
 >(`${PROPOSALS_ACTIONS_BASE}/${ProposalsActions.FetchProposals}`, async () => {
-  const { rpc } = chains[defaultChainId];
+  const { rpc } = chains.namada;
```

Comment:
In some cases, we are getting the Namada chain config from Redux state, and in others using `chains.namada`. What is the difference?

Reply:
oh, this is wrong! That should be pulled from Redux state. I can add it here, or in a follow up PR. I mainly focused on getting everything confirmed to work at least for transfers (to confirm extension is correct), but yeah, this needs to use Redux state here.

Reply:
I just pushed up a fix for this, and also the Staking actions were referring to default config instead of the Redux state